ci-operator/step-registry/ingress/gatewayapi/conformance-tests/ingress-gatewayapi-conformance-tests-commands.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "Go version: $(go version)"
+cd /go/src/github.com
+mkdir kubernetes-sigs && cd kubernetes-sigs
+# currently CRD v1.0.0 is supported
+git clone --branch release-1.0 https://github.com/kubernetes-sigs/gateway-api
```

Comment:
It is good that the CRD version is explicit.  However, I wonder whether it would be better and practical to specify the version in the openshift/cluster-ingress-operator repository.  We might eventually want to use this same job for multiple release branches with various CRD versions, and anyway, it would make version bumps easier if the CRD version for conformance tests and the CRD manifests were both in the same repository (namely openshift/cluster-ingress-operator).  

If it isn't easy to read the CRD version from another repository, then we can leave this as a follow-up.

Reply:
I suppose the simplest thing would be to add a `Makefile` target in openshift/cluster-ingress-operator for running the conformance tests.

Reply:
Good suggestion, let me try if I can make it with `Makefile`.